poker-pals/src/components/navbars/Navbar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <nav>
+        <b-navbar>
+            <b-navbar-brand href="#" class="mx-3">Poker Pals</b-navbar-brand>
```

Comment:
I think `href="#"` might have some weird side affects like adding a # to the url without actually doing anything